knexfile.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
 
 	staging: {
 		client: "pg",
-		connection: process.env.RDS_HOSTNAME,
+        connection: connect(process.env.DB),
```

Comment:
Formatting changed here after the env var pointer was updated. Please reformat back to inline tabs.